src/components/select/select.ts

```diff
@@ -365,6 +367,20 @@ export class Select extends BaseInput<string[]> implements AfterViewInit, OnDest
     });
   }
 
+  /**
+   * Close the select interface.
+   */
+  close() {
+    if (!this._overlay || !this.isFocus()) {
+      return;
+    }
+
+    let dismissPromise = this._overlay.dismiss();
+
+    dismissPromise.then(() => this._overlay = undefined);
```

Comment:
since this is not the only code path that can lead to the close of the overlay, I think this is not the right place to set this._overlay to undefined.